candidateport.py

```diff
@@ -0,0 +1,31 @@
+from candidate import cdetails
+class CandidateManager(cdetails):
```

Comment:
class CandidateManager:
	
	def __init__(self, file_manager):
		self._file_manager = file_manager

	def valid(self, candidate):
		return candidate.is_valid()

	def add(self, candidate):
		file_manager.save(candidate)

	def list(self):
		return file_manager.readAll()

	def  get(self, candidate_id)
	   candidates = list()
	   for(candidate in candiates):
	   		if candidate.id == candidate_id	
	   			return candiate
	def del()

Reply:
class FileManager:

	def __init__(self, file_name)

	def save()

	def list()

class CandidateFileManager(FileManager):

Reply:
appear.in/shubham